solr/solr-ref-guide/modules/configuration-guide/pages/caches-warming.adoc

```diff
@@ -86,7 +86,6 @@ The async cache provides most significant improvement with many concurrent queri
 However, the async cache will not prevent data races for time-limited queries, since those are expected to provide partial results.
 
 All caches can be disabled using the parameter `enabled` with a value of `false`.
-Caches can also be disabled on a query-by-query basis with the `cache` parameter, as described in the section xref:query-guide:common-query-parameters.adoc#cache-local-parameter[cache Local Parameter].
```

Comment:
This sentence is misleading, as only the filter cache inserts/lookups are affected by cache=false query parameter

Reply:
@epugh The doc does say that caches can be toggled in solrconfig.xml:
```
All caches can be disabled using the parameter `enabled` with a value of `false`.
```

But the **`cache` local parameter** only works with the filter cache, - so I removed all mentions of that parameter from the doc, and only kept one mention in the `Filter Cache` section (which redirects to Common Query Parameters).